src/ESPEasy.ino

```diff
@@ -462,6 +462,39 @@ struct ControllerSettingsStruct
   char          HostName[65];
   char          Publish[129];
   char          Subscribe[129];
+
+  IPAddress getIP() const {
+    IPAddress host(IP[0], IP[1], IP[2], IP[3]);
+    return host;
+  }
+
+  String getHost() const {
+    if (UseDNS) {
+      return HostName;
+    }
+    return getIP().toString();
+  }
+
+  boolean connectToHost(WiFiClient &client) {
+    if (UseDNS) {
+      return client.connect(HostName, Port);
```

Comment:
could just do: `return client.connect(getHost(), Port); ` without the if-construction ?

Reply:
There is an implementation for connect using IP and one using a string with hostname.
The string may also be a formatted IP address, but that will result in extra parsing. Just checking a boolean seemed faster to me :)